modules/ROOT/pages/salesforce/salesforce-connector-100-upgrade-migrate.adoc

```diff
@@ -0,0 +1,228 @@
+= Salesforce - Upgrade and Migrate - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../../assets/images/
+
+Upgrade Anypoint Connector for Salesforce (Salesforce Connector) to version 10.0.0.
+
+== Supported Upgrade Paths
+
+[%header,cols="50a,50a"]
+|===
+|From Version | To Version
+|9.8.x |10.0.0
+|===
+
+== Changes in This Release
+
+This release contains the following changes:
+
+* Removed deprecated connection types
+* Made the following changes to some operations:
+ ** Merged multiple operations into one
+ ** Renamed some operations and parameters
+ ** Changed some operation return types
+* Redesigned the Replay ID functionality
+* Changed the Mule TLS Context usage
+* Improved the error types
+* Removed the dependency on the Dozer library
+* Removed the keep alive feature as a configurable option. The connector now automatically performs one API call every
```

Comment:
Keep list items to one frag or sent:

* Removed the keep alive feature as a configurable option. 
The connector now automatically performs one API call every....

Reply:
[[Here is one change in a closed file vs. open: have to copy and paste text; no autoreplicate function]]